SKPhotoBrowser/SKZoomingScrollView.swift

```diff
@@ -186,6 +186,7 @@ open class SKZoomingScrollView: UIScrollView {
             photo.loadUnderlyingImageAndNotify()
         } else {
             indicatorView.stopAnimating()
+            photo.loadUnderlyingImageAndNotify()
```

Comment:
Unfortunately this breaks zooming for me. Every time I zoom the image, this `displayImage` method is called, and this call to `photo.loadUnderlyingImageAndNotify()` seems to reset the zoom back to default.

Reply:
Actually it's worse than this - this line produces an infinite callback loop which causes `displayImage` to be called continuously, thus constantly re-setting zoom.

The actual root of the problem is that the `SKPhoto` class doesn't provide any way to tell the difference between a placeholder image vs full image being loaded. The `SKZoomingScrollView` incorrectly assumes that the full image has loaded when the `SKPhoto.underlyingImage` property is not `nil` -  https://github.com/suzuki-0000/SKPhotoBrowser/blob/8e42c4211bc554559cde8b91838da425aa408032/SKPhotoBrowser/SKZoomingScrollView.swift#L16-L19